products/green_fast_track/models/intermediate/flags/int_flags__zoning.sql

```diff
@@ -2,50 +2,69 @@ WITH pluto AS (
     SELECT * FROM {{ ref('stg__pluto') }}
 ),
 
-zoning_districts_exploded AS (
+zoning_districts AS (
+    SELECT * FROM {{ ref('int__zoning_districts') }}
+),
+
+district_mappings AS (
+    SELECT * FROM {{ ref('zoning_district_mappings') }}
+),
+
+district_categories AS (
+    SELECT * FROM {{ ref('zoning_district_categories') }}
+),
+
+districts_mapped AS (
     SELECT
-        bbl,
-        UNNEST(ARRAY[zonedist1, zonedist2, zonedist3, zonedist4]) AS zd
-    FROM pluto
+        zoning_districts.bbl,
+        zoning_districts.zoning_district_type,
+        district_mappings.category_type
+    FROM zoning_districts
+    LEFT JOIN district_mappings ON zoning_districts.zoning_district_type = district_mappings.zoning_district_type
 ),
 
-zoning_districts_split_exploded AS (
+lot_with_zoning_categories AS (
```

Comment:
This is maybe just personal preference, but I would do the group by in the last query - both are very short and readable, and one more line will still be quite legible

Reply:
But at a minimum, I will again protest the `GROUP BY 1`, just seems to be inviting error down the road